src/wx/guiinit.cpp

```diff
@@ -2973,17 +2974,14 @@ bool MainFrame::BindControls()
 
                 // store checkable items
                 if (mi->IsCheckable()) {
-                    checkable_mi_t cmi = { cmdtab[i].cmd_id, mi, NULL, NULL, 0, 0 };
+                    checkable_mi_t cmi = { cmdtab[i].cmd_id, mi, 0, 0 };
                     checkable_mi.push_back(cmi);
 
-                    for (int j = 0; j < num_opts; j++) {
-                        wxString menuName = wxString(opts[j].cmd, wxConvUTF8);
-
-                        if (menuName == cmdtab[i].cmd) {
-                            if (opts[j].intopt)
-                                MenuOptionIntMask(opts[j].cmd, *opts[j].intopt, (1 << 0));
-                            else if (opts[j].boolopt)
-                                MenuOptionBool(opts[j].cmd, *opts[j].boolopt);
+                    for (const VbamOption& option : VbamOption::AllOptions()) {
+                        if (option.is_int()) {
```

Comment:
What about `double`? Or is that not relevant here?

Reply:
I don't really understand what this code is doing to be honest. The old code was not using `double` so I didn't add it here either.
I will revisit when I have a better understanding of what is happening here.

Reply:
I separately spent some time on code that makes use of this so I understand what is happening here better. This code initializes the checkable menu items to be checked or not and then spews a warning if we missed one - that's what the `initialized` field handles. So I believe we don't need to handle `double` here.